src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIOffenderService.kt

```diff
@@ -158,6 +158,9 @@ class CommunityAPIOffenderService(
       }
     }
 
+    if (responsibleOfficers.isNullOrEmpty()) {
```

Comment:
Does it make sense to have single return 
```suggestion
    return responsibleOfficers?.firstOrNull()
```

Reply:
Good spot, thanks Amol!